src/content/docs/en/core-concepts/astro-pages.mdx

```diff
@@ -25,7 +25,13 @@ A single file can also generate multiple pages using [dynamic routing](/en/core-
 
 ### Link between pages
 
-Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your Astro pages to link to other pages on your site.
+Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your Astro pages to link to other pages on your site. Use a **URL path relative to your root domain** as your link, not a relative file path.
+
+For example, to link to `https://example.com/authors/sonali/` from any other page on `example.com`:
+
+```astro title="src/pages/index.astro"
+Read more <a href="/authors/sonali/">about Sonali</a>.
```

Comment:
```suggestion
Read more <a href="/authors/sonali">about Sonali</a>.
```

Reply:
I thought about that! It's probably not *required*, but for example in Astro docs it *is* required. So I think it's safer to include it, because it won't be wrong either way!

Thank you for checking!  💜